Towny/src/main/java/com/palmergames/bukkit/towny/object/Resident.java

```diff
@@ -881,6 +881,20 @@ public Collection<TownBlock> getTownBlocks() {
 		return Collections.unmodifiableCollection(townBlocks);
 	}
 
+	/**
+	 * @return All towns that the resident is outlawed in
+	 */
+	public List<Town> getTownsOutlawedIn() {
```

Comment:
This doesn't really need to exist as a method imo, this can just be done in the ResidentCommand method.

Reply:
I could see other parts of the code being refactored to use this though.